templates/Dockerfile-rpm.tmpl

```
@@ -105,6 +105,28 @@ RUN [[ %%os%% != centos ]] || [[ %%release%% != 7 ]] || ( \
     yum -y install git && \
     yum -y install  devtoolset-8-gcc-c++ && scl enable devtoolset-8 bash )
 
+# install python 3.8 to be able to execute tools scripts
+# if os in (centos, oraclelinux) and release in (7, 8)
+RUN !( ( [ %%os%% = centos ] && [ %%release%% = 7 ] ) || \
+       ( [ %%os%% = centos ] && [ %%release%% = 8 ] ) || \
+       ( [ %%os%% = oraclelinux ] && [ %%release%% = 7 ] ) || \
+       ( [ %%os%% = oraclelinux ] && [ %%release%% = 8 ] ) \
```

```suggestion
RUN !( ( [ %%os%% = centos ] && [ %%release%% <= 8 ] ) || \
       ( [ %%os%% = oraclelinux ] && [ %%release%% <= 8 ] ) \
```

this is easier to read IMO. this is optional and you can ignore this comment.